docs/src/atomistic/reference/openmm.rst

```diff
@@ -0,0 +1,39 @@
+OpenMM integration
+==================
+
+.. py:currentmodule:: metatensor.torch.atomistic
+
+:py:mod:`openmm_interface` contains the ``get_metatensor_force`` function,
+which can be used to load a :py:class:`MetatensorAtomisticModel` into an
+``openmm.Force`` object able to calculate forces on an ``openmm.System``.
+
+.. autofunction:: metatensor.torch.atomistic.openmm_force.get_metatensor_force
+
+In order to run simulations with ``metatensor.torch.atomistic`` and ``OpenMM``,
+we recommend installing ``OpenMM`` from conda, using
+``conda install -c conda-forge openmm-torch nnpops``. Subsequently,
+metatensor can be installed with ``pip install metatensor[torch]``, and a minimal
```

Comment:
Since there is #665, should we recommend building metatensor-torch from sources for now? Something like ``pip install --no-binary=metatensor-torch metatensor[torch]``

Reply:
Sounds good